diff --git a/node_modules/gl/binding.gyp b/node_modules/gl/binding.gyp
index ab60fbf..e3c319b 100644
--- a/node_modules/gl/binding.gyp
+++ b/node_modules/gl/binding.gyp
@@ -42,7 +42,7 @@
               'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',
               'MACOSX_DEPLOYMENT_TARGET':'10.8',
               'CLANG_CXX_LIBRARY': 'libc++',
-              'CLANG_CXX_LANGUAGE_STANDARD':'c++17',
+              'CLANG_CXX_LANGUAGE_STANDARD':'c++20',
               'GCC_VERSION': 'com.apple.compilers.llvm.clang.1_0'
             },
         }],
